data/json/items/book/firstaid.json

```diff
@@ -151,7 +151,7 @@
     "name": { "str": "Merck Veterinary Manual", "str_pl": "copies of Merck Veterinary Manual" },
     "description": "This expansive text covers 'all domestic species and diseases in veterinary medicine worldwide'.  Should you need information regarding zoonotic diseases, off-label use of medication for animals, common dysfunctions and diseases of animals, or any other veterinary reference info, this is your book.  There are some interesting new chapters on animal cloning in this edition.",
     "weight": "5790 g",
-    "volume": "500 ml",
+    "volume": "6 L",
```

Comment:
When I looked this up on amazon, and converted the dimensions to liters I got 2.16 L

Reply:
Very likely. I was not aware this was a real book, what I did is I took one of my larger medical books off my shelf and tentatively calculated its volume. I will change it to 2.16l in a second.